Content.Server/GameObjects/VisibilityFlags.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Content.Server.GameObjects
+{
+    [Flags]
+    public enum VisibilityFlags
+    {
+        None = 0,
+        Normal = 1,
```

Comment:
This isn't used so probably best to remove it to avoid confusion.

Reply:
Done